iam-policy-secretsmanager.tf

```diff
@@ -14,7 +14,7 @@ resource "aws_iam_role_policy" "secretsmanager_for_ecs_readonly" {
     "Statement": {
         "Effect": "Allow",
         "Action": "secretsmanager:GetSecretValue",
-        "Resource": [ "${join("\",\"",concat(data.aws_secretsmanager_secret.db_ro_user.*.arn,data.aws_secretsmanager_secret.ldap_user.*.arn,data.aws_secretsmanager_secret.ranger_audit.*.arn))}" ]
+        "Resource": [ "${join("\",\"",concat(data.aws_secretsmanager_secret.db_ro_user.*.arn,data.aws_secretsmanager_secret.ldap_user.*.arn,data.aws_secretsmanager_secret.ranger_audit.*.arn,data.aws_secretsmanager_secret.docker_registry.*.arn))}" ]
```

Comment:
we may not need this change.

Reply:
@rpoluri  - we may need later if we ever need to fetch some binaries from artifactory from docker container.
if you think we may not need this in near future then i can remove this.

Reply:
we shouldn't be doing this, instead containers should be just replaced.

Reply:
@rpoluri  updated PR